operations/producer/producer-operation.go

```diff
@@ -125,6 +127,15 @@ func (operation *ProducerOperation) Produce(topic string, flags ProducerFlags) {
 		// print an empty line that will be replaced when updating the status
 		output.Statusf("")
 
+    var rl ratelimit.Limiter
+
+		if flags.RateInSeconds == -1 {
+			output.Infof("No rate limiting was set, running without constraints")
```

Comment:
please use `output.Debugf` so that the message is only printed when the command is executed with `-V`